README.md

```diff
@@ -201,16 +152,6 @@ __PACKAGE__->meta->make_immutable;
 1;
 ```
 
-### Never override new
```

Comment:
This is a common sense thing that is already well documented in Moose. I don't think it is very valuable for us to reproduce in our style guide.